connector-runtime/connector-runtime-spring/src/main/java/io/camunda/connector/runtime/inbound/importer/ProcessDefinitionInspector.java

```diff
@@ -102,16 +102,25 @@ private List<InboundConnectorDefinitionImpl> inspectBpmnProcess(
     List<BaseElement> inboundEligibleElements =
         INBOUND_ELIGIBLE_TYPES.stream()
             .flatMap(type -> process.getChildElementsByType(type).stream())
+            .filter(
+                element -> {
+                  Map<String, String> zeebeProperties = getRawProperties(element);
+                  if (zeebeProperties != null && zeebeProperties.get("inbound.type") != null) {
```

Comment:
nit: you can reference `Keywords.INBOUND_TYPE_KEYWORD`

Reply:
You are right, we should avoid these kind of magic strings, fixed it 👍